UnitTests/Sources/BugReportServiceTests.swift

```diff
@@ -19,7 +19,13 @@ import Foundation
 import XCTest
 
 class BugReportServiceTests: XCTestCase {
-    let bugReportService = MockBugReportService()
+    var bugReportService: BugReportServiceProtocolMock!
+
+    override func setUpWithError() throws {
+        bugReportService = BugReportServiceProtocolMock()
+        bugReportService.underlyingCrashedLastRun = false
+        bugReportService.submitBugReportProgressListenerReturnValue = SubmitBugReportResponse(reportUrl: "https://www.example/com/123")
```

Comment:
I'm guessing this return value isn't actually checked?
```suggestion
        bugReportService.submitBugReportProgressListenerReturnValue = SubmitBugReportResponse(reportUrl: "https://www.example.com/123")
```